### PR TITLE
ba.json

### PR DESCRIPTION
**Pull request by 2blqck**
I added a Bosnian translation and made minor changes to the English translation.

### CODE REVIEW

File: ba.json [hunk 1]
```diff
@@ -0,0 +1,1805 @@
...
+            "FACE_ID_PASSWORD": "Prepoznavanje lica i šifra",
+            "FACE_ID_PASSWORD_DESCRIPTION": "Zaštiti svoj telefon",
+            "ENTER_PASSCODE": "Odaberi šifru",
+            "PASSCODE": "Šifra",
+            "FACE_ID": "Prepoznavanje lica",
+            "RESET_FACE_ID": "Resetuj prepoznavanje lica",
+            "ENABLE_FACE_ID": "Uključi prepoznavanje lica",
+            "PIN_CODE": "Pin",
+            "PIN_CODE_ERROR": {
+                "TITLE": "Error",
```
Comment (Maximus7474) [inline ba.json:126]: Added PIN_CODE_ERROR section, needs translating.

File: ba.json [hunk 2]
```diff
@@ -0,0 +1,1805 @@
...
+                }
+            }
+        },
+        "WALLET": {
+            "TITLE": "Novčanik",
+            "BALANCE": "Stanje",
+            "SEND": "Pošalji",
+            "NEXT": "Dalje",
+            "DONE": "Gotovo",
+            "DONE_SUBTITLE": "You sent {amount} to {number}",
```
Comment (Maximus7474) [inline ba.json:653]: Missing translations on lines:
- 653, 654, 655
- 657, 658, 659, 660

File: ba.json [hunk 3]
```diff
@@ -0,0 +1,1805 @@
...
+            "PRIVATE_ACCOUNT": "Ovaj nalog je privatan",
+            "PRIVATE_ACCOUNT_DESCRIPTION": "Zapratite ovaj nalog da vidite objave i storye.",
+            "ERROR": "Greška",
+            "USERNAME_TAKEN": "Ime je već zauzeto",
+            "UNKNOWN": "Nepoznata greška",
+            "UNKNOWN_ACCOUNT": "Nalog ne postoji",
+            "INVALID_USERNAME": "Neispravano ime",
+            "INCORRECT_PASSWORD": "Neispravna šifra",
+            "OK": "OK",
+            "LOCATION": "Location",
```
Comment (Maximus7474) [inline ba.json:1117]: Added missing field LOCATION & CHOOSE_LOCATION, requires translation.

**Review by Maximus7474 — CHANGES_REQUESTED** [review]
Some missing translations have been added if you could update it.

**Comment by Maximus7474** [comment]
Hi @2blqck ,

Thanks for your contribution, I've left some comments if you could check them out

**Comment by 2blqck** [comment]
I updated it, take a look now.

**Review by Maximus7474 — APPROVED** [review]

**Comment by Maximus7474** [comment]
Thanks for the update, really appreciated !

**Pull request merged**